nibabel/deprecated.py

```diff
@@ -63,3 +66,18 @@ def __init__(self, *args, **kwargs):
                       FutureWarning,
                       stacklevel=2)
         super(FutureWarningMixin, self).__init__(*args, **kwargs)
+
+
+class VisibleDeprecationWarning(UserWarning):
+    """ Deprecation warning that will be shown by default
+
+    Python >= 2.7 does not show standard DeprecationWarnings by default:
+
+    http://docs.python.org/dev/whatsnew/2.7.html#the-future-for-python-2-x
+
+    Use this class for cases where we do want to show deprecations by default.
+    """
+    pass
+
+
+deprecate_with_version = Deprecator(cmp_pkg_version)
```

Comment:
I wasn't planning on it.  Numpy uses these for deprecations where the user is probably making a coding error.  We could promote all our deprecations, but that might be noisy.